packages/ui/src/components/organisms/ProductGrid/styles.scss

```
@@ -21,19 +21,17 @@
-      margin-top: var(--fs-product-grid-gap-desktop);
-    }
+    padding-bottom: var(--fs-product-grid-gap-tablet);
   }
 
-  @include media("<notebook") { padding-bottom: var(--fs-grid-gap-2); }
+  @include media("<notebook") {
+    padding-bottom: var(--fs-product-grid-gap-mobile);
+  }
 
   @include media(">=notebook") {
     grid-gap: var(--fs-product-grid-gap-desktop);
```

Stylelint (17.11.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)